virus/Antagonascensor_sororminor_and_sorormajor.md

```diff
@@ -0,0 +1,291 @@
+# {Q1A Paper Title}
```

Comment:
Add paper title here, anything in squiggle brackets are descriptions that should be updated (and brackets removed)